csclient/csclient.go

```diff
@@ -80,8 +80,12 @@ func New(p Params) *Client {
 	}
 }
 
+// ErrNoInteraction is the error cause returned by the default Params.VisitWebPage
+// value when it is nil.
+var ErrNoInteraction = errgo.New("interaction required but no web browser configured")
```

Comment:
"improper web browser configuration, authorisation process can not be completed." maybe?